### DISCUSSION TITLE
supabase login not accepting access token

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#29925] [Questions]
I am running "supabase login" from a terminal, docker is running fine.  But at login it is asking for access token, but I cant paste or type anything in to the prompt for the access token, if I just hit return I get an error: "Invalid access token format. Must be like `sbp_0102...1920`.  "

I have access tokens setup, but cannot enter anything, it used to work beforehand.  Any ideas?

**Comment by dannasc** [comment]
I'm having the same problem.

**Comment by Pinwheeler** [comment]
Why is this closed?

**Comment by thepearl** [comment]
same

**Comment by jao-o-garcia** [comment]
In your terminal do the following:
export SUPABASE_ACCESS_TOKEN=`personal access token (sbp_*****)`
npx supabase link --project-ref `project-id`
npx supabase login

**Reply by Cabezaa** [reply]
Thanks a lot, @jaogarcia97 . I went crazy searching for a possible solution for Windows + PowerShell, and that solution works! (i created a personal token from the Supabase access token dashboard and then used it as an ENV)
